Makefile

```diff
@@ -132,6 +132,10 @@ build/addon/data/panel-bundle.js: $(shoot_panel_dependencies)
 	@mkdir -p $(@D)
 	# Save the bundle dependencies:
 	browserify --list -e ./build/addon/data/shoot-panel.js | sed "s!$(shell pwd)/!!g" > build/shoot-panel-dependencies.txt
+	cat ./build/addon/lib/shooter.js | ./bin/_fixup_panel_js > ./build/addon/lib/shooter-new.js
+	mv ./build/addon/lib/shooter-new.js ./build/addon/lib/shooter.js
```

Comment:
Another thing to think about related to this is that I don't think shooter.js and shoot-panel.js need to be in the xpi at all -- they are included in the output of browserify.